code/modules/mob/living/living.dm

```diff
@@ -2257,6 +2257,9 @@ GLOBAL_LIST_EMPTY(fire_appearances)
 /mob/living/proc/is_face_visible()
 	return TRUE
 
+/// Sprite to show for photocopying mob butts
+/mob/living/proc/get_butt_sprite()
+	return
```

Comment:
```suggestion
	return null
```
 Just to make it abundantly obvious that this will return either a string or null.